src/markdown-pages/docs/cli-command-reference/installing-cli-tool.md

```diff
@@ -0,0 +1,66 @@
+---
+path: "/docs/cli-command-reference/installing-cli-tool"
+title: "Installing the Galasa CLI"
+---
+
+This section provides details about prerequisite software requirements and information about how to download and install the Galasa command line interface tool (Galasa CLI) on your local machine. 
+
+## Prerequisites
+
+| Software |  Description  |
+| :---- | :-------- | 
+| Java JDK  | Required. Galasa tests and Managers are written in Java - you need to install a Java version 11 JDK or later to use it. _Note:_ We do not currently support Java 17 or later. |
+| Maven or Gradle  | You must install either Maven or Gradle in order to build Galasa projects. Galasa projects are hierarchical file structures that provide the ability to store and run Galasa tests. All Galasa versions are compatible with Gradle releases 6.8.x and later. |
+| 3270 emulator | Optional. Although you do not need a 3270 emulator to run a Galasa test (even if it tests a 3270 application) you can use one to explore Galasa Simbank, a simulated version of an application that helps you get acquainted with Galasa before connecting to a real mainframe to run your own tests. There are many such emulators available but IBM's Personal Communications (PCOMM) is frequently used, as is IBM's Host on Demand software, which includes support for Windows, Linux and MacOS.| 
+
+
+## Downloading the Galasa CLI
+
+To install Galasa for using in the command-line you first need to download the binary file for the Galasa CLI from the Galasa cli respository in GitHub. 
+
+The following versions of the Galasa CLI are available to download for different operating systems and machine architectures:
+
+| Operating system  |  Download  |
+| :---- | :-------- | 
```

Comment:
Maybe we should have a middle column 'Architecture` so we have arm64 or 64-bit x86 ? It's missing information on the two MaxOSX entries.